### PR TITLE
    qa/tasks/ceph_fuse: fix ceph-fuse mount failed

### PR DESCRIPTION
**Pull request by LianneWang**
Fixes: http://tracker.ceph.com/issues/22205

    Signed-off-by: Lianne Wang <liyan.wang@xtaotech.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
could you add a "`Signed-off-by`" line at the end of your commit message? "git commit -s" will do the trick for you. see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#1-sign-your-work

File: qa/tasks/ceph_fuse.py [hunk 1]
```diff
@@ -103,6 +105,23 @@ def task(ctx, config):
     all_mounts = getattr(ctx, 'mounts', {})
     mounted_by_me = {}
 
+    # make sure someone is in active
+    mds_cluster = MDSCluster(ctx)
+    log.info('Wait for all MDSs to reach steady state...')
```
Comment (batrick) [inline qa/tasks/ceph_fuse.py]: You can simplify this with:
````
fs = Filesystem(ctx, name="cephfs")
fs.wait_for_daemons()
````

**Comment by LianneWang** [comment]
@batrick  @tchaikov

**Comment by LianneWang** [comment]
@tchaikov @batrick 

This has nothing to do with my submission：

99% tests passed, 1 tests failed out of 149

Total Test time (real) = 3600.23 sec

The following tests FAILED:
	  5 - smoke.sh (Timeout)
Errors while running CTest
rm: cannot remove ‘/tmp/ceph-asok.10328’: Is a directory
rm: cannot remove ‘/tmp/ceph-asok.11799’: Is a directory
Build step 'Execute shell' marked build as failure
[BFA] Scanning build for known causes...
..[BFA] Found failure cause(s):
[BFA] No such file or directory from category Development
[BFA] Test failure
[BFA] Done. 2s
Setting status of 0a752e634a64baa1b678018f39a788be0284106d to FAILURE with url https://jenkins.ceph.com/job/ceph-pull-requests-arm64/9398/ and message: 'make check failed
 '
Using context: make check (arm64)
Finished: FAILURE

**Comment by tchaikov** [comment]
retest this please.

**Pull request merged**